Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -119,12 +119,18 @@
 
 
   <Furniture objectType="Power Generator">
-    <Name>Power Generator</Name>
-    <MovementCost>10</MovementCost>
-    <Width>2</Width>
-    <Height>2</Height>
+		<Name>Power Generator</Name>
+		<MovementCost>10</MovementCost>
+		<Width>2</Width>
+		<Height>2</Height>
 
-    <PowerGenerator supply="5"/>
+        <BuildingJob jobTime="0">
+        </BuildingJob>
+
```

Comment:
You might want to change these tabs to spaces, since we decided to indent with four spaces equalling a tab.